src/content/app/genome-browser/hooks/useGenomeBrowser.ts

```diff
@@ -186,9 +204,20 @@ const useGenomeBrowser = () => {
             ? trackStateForLabels.on.push(trackId)
             : trackStateForLabels.off.push(trackId);
 
-          config.showSeveralTranscripts
-            ? trackStateForSeveralTranscripts.on.push(trackId)
-            : trackStateForSeveralTranscripts.off.push(trackId);
+          /**
+           * TODO: think what to do about the saved state of the "several transcripts" toggle
```

Comment:
Do we need a separate ticket for this?

Reply:
Well, we do need to figure out what should happen when the 1/5 toggle has been toggled and then the page is reloaded. A separate ticket would be good.